camel-core/src/main/java/org/apache/camel/ComponentVerifier.java

```diff
@@ -253,56 +254,56 @@ static Attribute asAttribute(String attribute) {
         /**
          * Standard set of error codes
          */
-        enum StandardCode implements Code {
+        interface StandardCode extends Code {
```

Comment:
Here's now the change. tbh, I liked the enum code better:

* shorter
* It's sealed. Here, everybody could extend `StandardCode` to add own codes. enums can now be extended, though.

Also some people consider interfaces with only constants as an anti-pattern. Personally I dont mind much if camel already uses this pattern. Butting having interfaces only for constants feels a bit weird (well, thats what enums are for ;-)